pydarn/plotting/grid.py

```diff
@@ -178,11 +160,9 @@ def plot_grid(cls, dmap_data: List[dict], record: int = 0,
             warnings.simplefilter("ignore")
             for stid in dmap_data[record]['stid']:
                 _, aacgm_lons, _, _, ax =\
-                        Fan.plot_fov(stid, dtime,
-                                     boundary=fov,
+                        Fan.plot_fov(stid, date,
                                      lowlat=lowlat,
                                      line_color=line_color,
```

Comment:
```suggestion
                        Fan.plot_fov(stid, date,
```
Sorry, I didn't explain it very well. lowlat, line_color, radar_location, and radar_label are all in kwargs, so if they're named here and you don't give them as a kw in the call to plot it will break. Just need to remove them and add **kwargs